apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -218,6 +220,44 @@ const Invoice = new GQLListSchema('Invoice', {
             type: 'DateTimeUtc',
         },
 
+        acquiringIntegrationId: {
+            schemaDoc: 'Integration ID through which this invoice can be paid',
+            type: 'Virtual',
+            resolver: async (item, args, context) => {
+                const invoiceContext = await getById('InvoiceContext', item.context)
```

Comment:
The `InvoiceContext` model is removed. Please do not use it. See #4142.
It's better to add only one virtual field that emulates context:
```js
context { integration { hostUrl canGroupReceipts currencyCode } }
```
To find these fields you should find the `AcquiringIntegrationContext` for the organization with finished `invoiceStatus`.